optaplanner-8-benchmarks/optaplanner-perf-framework/data/nurserostering/long01.xml

```diff
@@ -460,458 +460,458 @@
           </ShiftDate>
           <DayOffRequest id="90">
-            <id>6</id>
+            <id>3</id>
             <employee reference="73"/>
             <shiftDate reference="83"/>
             <weight>1</weight>
           </DayOffRequest>
         </entry>
         <entry>
           <ShiftDate id="91">
-            <id>18</id>
-            <dayIndex>18</dayIndex>
-            <date>2010-01-19</date>
+            <id>16</id>
+            <dayIndex>16</dayIndex>
+            <date>2010-01-17</date>
             <shiftList id="92">
               <Shift id="93">
-                <id>90</id>
+                <id>80</id>
                 <shiftDate reference="91"/>
                 <shiftType reference="6"/>
-                <index>90</index>
-                <requiredEmployeeSize>8</requiredEmployeeSize>
+                <index>80</index>
+                <requiredEmployeeSize>6</requiredEmployeeSize>
               </Shift>
               <Shift id="94">
-                <id>91</id>
+                <id>81</id>
                 <shiftDate reference="91"/>
                 <shiftType reference="8"/>
-                <index>91</index>
-                <requiredEmployeeSize>8</requiredEmployeeSize>
+                <index>81</index>
+                <requiredEmployeeSize>6</requiredEmployeeSize>
               </Shift>
               <Shift id="95">
-                <id>92</id>
+                <id>82</id>
                 <shiftDate reference="91"/>
                 <shiftType reference="10"/>
-                <index>92</index>
-                <requiredEmployeeSize>5</requiredEmployeeSize>
+                <index>82</index>
+                <requiredEmployeeSize>3</requiredEmployeeSize>
               </Shift>
               <Shift id="96">
-                <id>93</id>
+                <id>83</id>
                 <shiftDate reference="91"/>
                 <shiftType reference="12"/>
-                <index>93</index>
-                <requiredEmployeeSize>6</requiredEmployeeSize>
+                <index>83</index>
+                <requiredEmployeeSize>4</requiredEmployeeSize>
               </Shift>
               <Shift id="97">
-                <id>94</id>
+                <id>84</id>
                 <shiftDate reference="91"/>
                 <shiftType reference="14"/>
-                <index>94</index>
-                <requiredEmployeeSize>2</requiredEmployeeSize>
+                <index>84</index>
+                <requiredEmployeeSize>1</requiredEmployeeSize>
               </Shift>
             </shiftList>
           </ShiftDate>
           <DayOffRequest id="98">
-            <id>9</id>
+            <id>0</id>
             <employee reference="73"/>
             <shiftDate reference="91"/>
             <weight>1</weight>
           </DayOffRequest>
         </entry>
         <entry>
-          <ShiftDate reference="3"/>
-          <DayOffRequest id="99">
-            <id>5</id>
+          <ShiftDate id="99">
```

Comment:
It looks like it used to be a reference to an existing instance and now it changes to a new instance. Why is that so?

Reply:
Yes, the references are not the same but I used the same as currently defined in OptaPlanner 8
So as far as the data set size stays the same the content could be completely different

Reply:
It may have an impact on the problem object and on the constraints. If a constraint compares object identity, assuming there is only a single instance of some fact, this change in the dataset might break it.